e2etests/sanity_test.go

```diff
@@ -0,0 +1,44 @@
+// +build e2e
+
+package e2etests
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	kubeLinterBinEnv = "KUBE_LINTER_BIN"
+)
+
+func TestKubeLinterWithBuiltInChecksDoesntCrashOnHelmChartsRepo(t *testing.T) {
+	kubeLinterBin := os.Getenv(kubeLinterBinEnv)
+	require.NotEmpty(t, kubeLinterBin, "Please set %s", kubeLinterBinEnv)
+
+	_, err := os.Stat(kubeLinterBin)
+	require.NoError(t, err)
+
+	tmpDir, err := ioutil.TempDir("", "")
+	require.NoError(t, err)
+	defer func() {
+		require.NoError(t, os.RemoveAll(tmpDir))
+	}()
+
+	chartsDir := filepath.Join(tmpDir, "charts")
+
+	gitCloneOut, err := exec.Command("git", "clone", "git@github.com:helm/charts.git", chartsDir).CombinedOutput()
+	require.NoError(t, err, "Git clone failed. output: %s", string(gitCloneOut))
+
+	kubeLinterOut, err := exec.Command(kubeLinterBin, "lint", chartsDir, "--config", "testdata/all-built-in-config.yaml").CombinedOutput()
+	// Something will fail for sure, so kube-linter will not return a success code.
```

Comment:
I think this is definitely a good start but should we consider removing this dependency to helm's repo? So that we can further verify the specific checks that failed.

Reply:
The reason I like this repo is that it has a very wide variety of charts with all sorts of configurations. This test is literally just to make sure that KubeLinter: a) finds _some_ lint errors in the charts and b) doesn't panic.

We should definitely add tests that take specific YAML files and verify the specific checks that failed. I'm working on that too, but this test will still be relevant because we get a huge amount of breadth for free from the Helm repo

Reply:
makes sense